src/Lib/Items/CanvasText.vala

```diff
@@ -57,7 +57,7 @@ public class Akira.Lib.Items.CanvasText : Goo.CanvasText, Akira.Lib.Items.Canvas
         // Force the generation of the item bounds on creation.
         Goo.CanvasBounds bounds;
         this.get_bounds (out bounds);
-
+        
```

Comment:
Added an accidentally removed line

Reply:
Now this empty line has blank characters.
Please remove them and leave a simple blank empty line.

Reply:
ok